src/library/baseplaylistfeature.cpp

```diff
@@ -650,22 +649,23 @@ QModelIndex BasePlaylistFeature::constructChildModel(int selected_id) {
     return m_childModel.index(selected_row, 0);
 }
 
-void BasePlaylistFeature::updateChildModel(int selected_id) {
-    buildPlaylistList();
+void BasePlaylistFeature::updateChildModel(int changedPlaylistId) {
+    // the following call fetches whole m_playlistList.
```

Comment:
The comments is outdated and doesn't really help:
- m_playlistList
- How updatePlaylistLabel() actually works should not matter here